<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>851</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="labelTitle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Panaderia La Quinta</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton0">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>620</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPrint">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>620</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLastTranVal">
      <property name="geometry">
       <rect>
        <x>424</x>
        <y>79</y>
        <width>241</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOpen">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>620</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Abrir Caja</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHistory">
      <property name="geometry">
       <rect>
        <x>614</x>
        <y>149</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Historial</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>190</y>
        <width>431</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton6">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>460</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOk">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>620</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">Background-color:rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>620</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">Background-color:rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;X</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButtonSum">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>680</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Resumir</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidgetTo">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>340</y>
        <width>392</width>
        <height>236</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidgetFrom">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>392</width>
        <height>236</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxFrom">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>610</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>12:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11:00 PM</string>
       </property>
      </item>
     </widget>
     <widget class="QListView" name="listViewSum">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>441</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTo">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>610</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>23</number>
      </property>
      <item>
       <property name="text">
        <string>01:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11:00 AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11:00 PM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12:00 AM</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelTotal">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>700</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTotalValue">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>700</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEdRec">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDelRec">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>60</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonGoRec">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>680</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Registro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPrintSum">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>690</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSumH">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>680</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Resumir / hora</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>491</width>
        <height>511</height>
       </rect>
      </property>
      <widget class="QCalendarWidget" name="calendarWidgetEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>392</width>
         <height>236</height>
        </rect>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>310</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonEdRec2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>390</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonGoAdmin">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>390</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>310</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Valor:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>272</x>
         <y>301</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
